akka-docs/src/test/java/jdocs/stream/operators/SinkDocExamples.java

```diff
@@ -105,6 +105,12 @@ static void lastOptionExample() {
     // #lastOption-operator-example
   }
 
+  static void foldExample() {
+    // #fold
+    Source.range(1, 100).runWith(Sink.fold(0, (res, number) -> res + number), system);
+    // #fold
```

Comment:
I think it would be good with another few lines actually consuming the result (that would also require that you catch the return value in a variable and make the type of the materialized value explicit.

For an existing operator sample that does this, you can look at Sink.last.